clowdapp.yaml

```diff
@@ -63,12 +63,12 @@ objects:
             value: ${DB_POOL_SIZE}
           - name: DB_MAX_OVERFLOW
             value: ${DB_MAX_OVERFLOW}
-    - name: processor
+    - name: inventory-events-processor
```

Comment:
Hello @digitronik - As @patilsuraj767  made changes in clowdapp file related to pods, does any change required in job template?

Reply:
As per my info, it will not affect.